packages/db/src/controller/level.ts

```diff
@@ -175,6 +177,17 @@ export class LevelDbController implements IDatabaseController<Uint8Array, Uint8A
 
     this.metrics?.dbWriteItems.inc({bucket}, itemsRead);
   }
+
+  /** Add collect function to capture metric for db size */
+  private addDbSizeMetricCollector(metrics: ILevelDbControllerMetrics): void {
+    const minKey = Buffer.from([0x00]);
+    const maxKey = Buffer.from([0xff]);
```

Comment:
I think the min and max keys are optional? Not setting = whole db?

Reply:
Omitting the start and end key causes an error `TypeError: The arguments 'start' and 'end' are required`, see [approximateSize input validation](https://github.com/Level/classic-level/blob/8a4717b2a2177ad8ad6b35963a61bd7c86bb279f/index.js#L95). The downstream calls also rely on those parameters to be set.

I am not 100% sure if the min and max key I am setting at the moment have the proper values to get the total db size. There is not explicit test in classic-level repository which tests that the full db size is returned by `approximateSize`.

Reply:
the specified `minKey` and `maxKey` seem to properly capture the whole db size